docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -226,7 +226,8 @@ For now `@TestHTTPResource` allows you to inject `URI`, `URL` and `String` repre
 
 [TIP]
 =====
-If `quarkus.servlet.context-path` is configured, make sure to prefix the endpoint with the provided value acting as a base path during injection of a `URI` through `TestHTTPResource` or when calling a rest endpoint with `RestAssured`
+If `quarkus.servlet.context-path` is configured, make sure to prefix the endpoint with the provided value acting as a base path
+during injection of a `URI` through `TestHTTPResource`.
```

Comment:
Can we automatically configure this too?

Reply:
Good point, I'll take a look at it tomorrow.

Reply:
It turns out the current `TestHTTPResourceManager` implementation has a bug which prevents the context path from being added to the URL. When several `QuarkusUnitTest` are run from the same project, the `TestHTTPResourceManager` static intializer is only called once when the first `QuarkusUnitTest` is run. Therefore, none of the configuration values (host and port in the current version) are correctly loaded.

@gsmet Is it OK if I try fixing the `TestHTTPResourceManager` issue in this PR? I don't know how far we are from `0.22.0`.